src/core/validation/README.md

```
@@ -0,0 +1,91 @@
+
+registry.addRule({
+  validatorType: 'custom',
+  shouldApply: () => true,
+});
+
+const engine = createValidationEngine(registry);
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)